src/AuditReader.php

```diff
@@ -237,7 +237,9 @@ public function find($className, $id, $revision, array $options = [])
         $columnMap = [];
 
         foreach ($classMetadata->fieldNames as $columnName => $field) {
-            $tableAlias = $classMetadata->isInheritanceTypeJoined() && $classMetadata->isInheritedField($field) && !$classMetadata->isIdentifier($field)
+            $tableAlias = $classMetadata->isInheritanceTypeJoined() && $classMetadata->isInheritedField(
+                $field
+            ) && !$classMetadata->isIdentifier($field)
```

Comment:
This reformatting is weird. I would prefer
```suggestion
            $tableAlias = $classMetadata->isInheritanceTypeJoined()
                && $classMetadata->isInheritedField($field)
                && !$classMetadata->isIdentifier($field)
```